spec/src/main/asciidoc/configuration.asciidoc

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2018 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// Contributors:
+// Emily Jiang
+//
+
+= MicroProfile Config and Service Mesh
+
+MicroProfile Config provides a solution to externalise the configuration. The default config sources includes environment varialbles, system properties and microprofile-config.properties on the classpath. 
+
+The properties defined in Kubernetes config map automatically transforms to environment variables. 
```

Comment:
Not automatically. Only when you specify

```
- envFrom:
        - configMapRef:
            name: <name of the config map>
```

Reply:
will update it.